code/__DEFINES/economy.dm

```diff
@@ -26,3 +26,43 @@
 #define ACCOUNT_SEC_NAME "Defense Budget"
 
 #define NO_FREEBIES "commies go home"
+
+//Monkestation edit begin
+/// How much mail the Economy SS will create per minute, regardless of firing time.
+#define MAX_MAIL_PER_MINUTE 3
+/// Probability of using letters of envelope sprites on all letters.
+#define FULL_CRATE_LETTER_ODDS 70
```

Comment:
Mail stuff shouldn't be in your bounty port

Reply:
I wound up copying the entire defines file from tg in a struggle to figure out what was missing, haha, I'll trim this up!